x-pack/plugins/alerting/server/rules_client/common/snooze_utils.ts

```diff
@@ -28,12 +28,12 @@ export function getSnoozeAttributes(
       snoozeSchedule: clearUnscheduledSnoozeAttributes(attributes),
     };
   }
+
   return {
     snoozeSchedule: (snoozeId
       ? clearScheduledSnoozesAttributesById(attributes, [snoozeId])
       : clearUnscheduledSnoozeAttributes(attributes)
     ).concat(snoozeSchedule),
-    muteAll: false,
```

Comment:
Just curious to know why this is being removed

Reply:
This prevents us from removing an indefinite snooze when adding a snooze schedule